collector.py

```diff
@@ -71,13 +79,23 @@ def collect_data(extract_options, current_session_name):
     print(REPORT)
 
 
+def save_report(sn):
+    db_store_path = ROOT_DIR + SEP + sn
+    file_path = db_store_path + SEP + 'report.txt'
+    REPORT.append(["Case Name", sn])
+    write_to_file(file_path, REPORT)
+
+
 if __name__ == '__main__':
     args = sys.argv
     args_set = set(args)
 
     if HELP_KEYS & args_set:
         print(help_str)
         sys.exit()
+    elif REPORT_GEN_KEYS & args_set:
```

Comment:
Can't we generate the report, along with data extraction? @Wolf-Legend

Reply:
For simplicity, I decided both feature to be separate. But we can add a flag in data extraction for generating report as well along side.